src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -73,6 +73,7 @@ public function getConfigTreeBuilder()
                 ->booleanNode('hide_user_not_found')->defaultTrue()->end()
                 ->booleanNode('always_authenticate_before_granting')->defaultFalse()->end()
                 ->booleanNode('erase_credentials')->defaultTrue()->end()
+                ->booleanNode('enable_authenticator_manager')->defaultFalse()->info('Enables the new Symfony Security system based on Authenticators, all used providers must support this before enabling this.')->end()
```

Comment:
This may be legit ... but also a typo, so please double check: `used providers` -> `user providers` ?

Reply:
"used providers" is legit, but to avoid confusion I'll change it to "used authenticators".